library/tiqr/Tiqr/UserSecretStorage/Pdo.php

```
@@ -36,13 +38,16 @@ class Tiqr_UserSecretStorage_Pdo extends Tiqr_UserStorage_Pdo implements Tiqr_Us
      *
      * @param array $config The configuration that a specific user class may use.
      */
-    public function __construct($config, $secretconfig = array())
+    public function __construct($config, LoggerInterface $logger, $secretconfig = array())
     {
+        $this->logger = $logger;
         $this->tablename = isset($config['table']) ? $config['table'] : 'tiqrusersecret';
         try {
             $this->handle = new PDO($config['dsn'],$config['username'],$config['password']);
         } catch (PDOException $e) {
-            return false;
```

Do we want to update the phpdoc of the functions that now throw instead of returning false?

---

My personal preference is to remove all PHPDoc that contributes no valuable explanation.

Some methods now throw an exception. And that might be a good thing to advertise on the PHPDoc!